fn/develop/configs.md

```diff
@@ -1,22 +1,115 @@
-# Function Config Vars
+# Use of environment variables
 
-There are three ways to get configuration variables into a function, all of which will be available as environment variables.
-These are ordered in order of preference, the later ones overriding the previous ones.
+If you want to use configuration variables into a function and make them available as environment variables, then you can set them using the Fn tool or the func.yaml file.
+ 
+By using the Fn tool, you can set configuration variables on the application or configuration level.
 
-## (1) Application level configuration
+# Set environment variables
 
+The following three examples shows you how to set the configuration variables on different levels using the `connection-app` application. These are listed in order of preference, the later ones overrides the previous ones:
+
+
+(1) Application level configuration
+ 
+```sh
+fn config app <app-name> <key> <value>
+```
+For example, set the `DB_URL` configuration variable with value of `jdbc:mysql`. 
 ```sh
-fn config app myapp LOG_LEVEL debug
+$ fn config app connection-app DB_URL jdbc:mysql
 ```
 
+(2) Function configuration from `func.yaml` file
 
-## (2) Function configuration from func.yaml
+Use the `func.yaml` file to define your configuration variables in the `config` key. For example: set the `DB_DRIVER` configuration variable with value of `mysqlDriver`.
 
+```sh
+schema_version: 20180708
+name: connection
+version: 0.0.16
+runtime: java
+build_image: fnproject/fn-java-fdk-build:jdk11-1.0.99
+run_image: fnproject/fn-java-fdk:jre11-1.0.99
+cmd: com.example.fn.Connection::getConnection
+config:
+  DB_DRIVER: mysqlDriver
+```
 See [Function file](func-file.md) for more info.
 
+(3) Function level configuration
 
-## (3) Function level configuration
+After you deploy your function, you can set configuration variables at the level configuration.
```

Comment:
Suggested change:

..., you can set function level configuration variables. The variables only apply to the specific function identified.

Reply:
Done